docs/stylesheets/extra.css

```
@@ -0,0 +1,822 @@
+  min-width: 300px;
+  padding-right: 2rem;
+  position: relative;
+  z-index: 1;
+  animation: fadeIn 0.8s ease-out forwards;
+}
+
+.hero-logo-title {
+  display: flex;
+  align-items: center;
+  animation-delay: 0.2s;
+  animation: fadeIn 0.8s ease-out 0.2s both;
```

Biome (1.9.4)

[error] Unexpected shorthand property animation after animation-delay



(lint/suspicious/noShorthandPropertyOverrides)